src/sif/autocost.cc

```diff
@@ -694,6 +696,7 @@ void ParseAutoCostOptions(const rapidjson::Document& doc,
   JSON_PBF_DEFAULT(co, false, json, "/include_hot", include_hot);
   JSON_PBF_DEFAULT(co, false, json, "/include_hov2", include_hov2);
   JSON_PBF_DEFAULT(co, false, json, "/include_hov3", include_hov3);
+  JSON_PBF_RANGED_DEFAULT(co, kVehicleSpeedRange, json, "/top_speed", top_speed);
```

Comment:
there was a previously unnoticed bug that prevented costing profiles to overwrite ranged defaults set in `dynamiccost`' base cost. Most likely affected `motorscootercost` as well